packages/karma-credential-loader/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@aws-sdk/client-sts": "file:../../clients/client-sts",
+    "@aws-sdk/client-sts": "*",
```

Comment:
this was originally to break a circular dependency, but there is, at least currently, no cycle related to this package